### PR TITLE
Add table of JVM to Unicode

### PR DESCRIPTION
**Pull request by gem-neo4j**
Unicode version taken form Java docs: https://docs.oracle.com/en/java/javase/22/docs/api/java.base/java/lang/Character.html

### CODE REVIEW

**Comment by gem-neo4j** [comment]
[GQL compliance - Whitespace](https://trello.com/c/PogUSak4/940-gql-compliance-whitespace)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-944.surge.sh

Updated pages:
- [Parsing](https://neo4j-docs-cypher-944.surge.sh/cypher-manual/5/syntax/parsing)

File: modules/ROOT/pages/syntax/parsing.adoc [hunk 1]
```diff
@@ -24,6 +24,17 @@ The following example escapes the unicode character `A` (`\u0041`) in the keywor
 M\u0041TCH (m) RETURN m;
 ----
 
+The Unicode Version used by Cypher is coupled with the JVM version that is running.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/syntax/parsing.adoc]:
```suggestion
The Unicode version used by Cypher depends on the running JVM version.
```

File: modules/ROOT/pages/syntax/parsing.adoc [hunk 2]
```diff
@@ -24,6 +24,17 @@ The following example escapes the unicode character `A` (`\u0041`) in the keywor
...
+
+[options="header", cols="1,2,3"]
+|===
+| Neo4j Version	| JVM compliancy                                   | Unicode Version
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/syntax/parsing.adoc]:
```suggestion
| Neo4j version	| JVM compliancy                                   | Unicode version
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-944.surge.sh

Updated pages:
- [Parsing](https://neo4j-docs-cypher-944.surge.sh/cypher-manual/5/syntax/parsing)

**Review by JPryce-Aklundh — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**